### DISCUSSION TITLE
Bug

### DISCUSSION THREAD

**Discussion by memaslr16** [Chatterino/chatterino2#3295] [Q&A] [answered]
There is a bug or problem with the chaterrino that every time I start it I Messager histoy service unavailable (Error 0)

**Comment by Felanbird** [comment, marked as answer]
> 
> 
> There is a bug or problem with the chaterrino that every time I start it I Messager histoy service unavailable (Error 0)

Chatterino fetches Message History from a 3rd party service - which happens to be down right now.
It will be back as soon as the service is fixed.
